source/edge-server/configure.txt

```diff
@@ -140,121 +140,106 @@ host.
-        manage the instance in a production environment. Valid values are 
-        either ``compose`` or ``kubernetes``.
+      .. include:: /includes/edge/edge-kube-install-procedure.rst
 
-      You can provide the following optional flags to specify additional
-      configuration details:
-
-      .. include:: /includes/edge/edgectl-init-flags.rst
-
-   .. step:: Supply the Edge Server Token
-
-      When first configuring the Edge Server instance, you're prompted to 
-      supply the Edge Server token. This is the token you got when you added 
-      the instance to your service.
+.. _start-and-stop-edge-server:
 
-      If you no longer have the token, you can generate a new token from the
-      :ref:`edge-server-details` in your service.
+Start and Stop the Edge Server
+------------------------------
 
-      The Edge Server instance exchanges this token for a secret. It uses the
-      secret to authenticate with Atlas when it syncs data.
+.. tabs::
 
-   .. step:: (Optional) Create a New User
+   .. tab:: Docker Compose
+      :tabid: docker
 
-      If you'd like to connect to the Edge Server instance using a MongoDB
-      Driver or tool, you must have a username/password credential, or bypass
-      authentication. If you already have a username/password credential, or 
-      don't plan to connect to the Edge Server instance using a MongoDB Driver
-      or tool, you can skip this step.
+      Use ``edgectl`` to start and stop the Edge Server instance.
 
-      For more information, refer to :ref:`edge-server-wireprotocol-connect`.
+      To start all Edge Server instance containers, use the ``start`` command:
 
-      If you don't already have a username/password credential, you can create
-      one by following the prompts during ``edgectl init``.
+      .. code-block:: shell
 
-      Otherwise, you can manually :ref:`create-email-password-user` later
-      using the App Services UI, the App Services CLI, or the App Services
-      Admin API.
+         edgectl start
 
-      Alternately, if you plan to implement an authorization flow later, you
-      can bypass authorization with the following command and flag:
+      To stop all Edge Server instance containers, use the ``stop`` command:
 
-      .. code-block:: console
+      .. code-block:: shell
 
-         edgectl config --insecure-disable-auth=true
+         edgectl stop
 
-.. _start-and-stop-edge-server:
+   .. tab:: Kubernetes
+      :tabid: kubernetes
 
-Start and Stop the Edge Server
-------------------------------
+      Kubernetes deployments do not have a direct analog to ``edgectl start``
+      and ``edgectl stop``.
 
-Use ``edgectl`` to start and stop the Edge Server instance.
+      Instead, use your deployment tool to appropriately stop or delete
+      the containers.
 
-To start all Edge Server instance containers, use the ``start`` command:
+.. include:: /includes/important-stop-edge-server.rst
```

Comment:
should we update this admonition _slightly_? something like "stop Edge Server using `edgectl stop` or your deployment tool..."?